keyboards/handwired/justageng/office60/readme.md

```diff
@@ -0,0 +1,127 @@
+# Office60
+
+![Office60](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Jia Geng](https://github.com/JiaGengChang)
+* Hardware Supported: STM32F103C8T6 (aka the Blue Pill)
+* Hardware Availability: Search "bluepill stm32" on aliexpress, and choose the unwelded version if possible since we will be soldering wires directly onto the board
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/justageng/office60:default
+
+Flashing example for this keyboard:
+
+    make handwired/justageng/office60:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader the following 2 ways:
+* **Bootmagic**: Hold down the escape key, or the key at the top left corner
+* **Physical reset button**: Briefly press the reset button on the bluepill 
+
+
+Windows 10 instructions for flashing firmware:
```

Comment:
Most of this looks to be just stm32duino flashing instructions, which are posted elsewhere.  

For instance: 

https://github.com/rogerclarkmelbourne/Arduino_STM32/wiki/Flashing-Bootloader-for-BluePill-Boards

Reply:
Do you prefer if I cut down on the length of the readme, and point users to other sites for instructions?

Reply:
I have added the link you provided to the readme for additional reference.